inst/Upgrades.qml

```diff
@@ -0,0 +1,289 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName: 		"BinomialTest"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
```

Comment:
Not too important: Is it just me, or do the tabs misalign? I see two tabs "fromVersion:" and three tabs "toVersion:"

Reply:
It's just GitHub screwing up the indentation, in Qt it looks fine:
![Screenshot 2022-10-04 at 13 17 09](https://user-images.githubusercontent.com/7093546/193805392-dbbaa037-3a73-4754-b6f3-f01e9ab79619.png)